tasks/users.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: "Make sure the MySQL users are present"
+  mysql_user:
+    name: "{{ item.name }}"
+    password: "{{ item.pass | default('pass') }}"
```

Comment:
May be breacking when there is no password set is not a bad idea...

Reply:
This is clone from: https://github.com/artefactual-labs/ansible-percona/blob/master/tasks/users.yml

So we can investigate it in a new issue and change both roles if needed.